### PR TITLE
chore: update issue templates

### PR DESCRIPTION
**Pull request by tarekoraby**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    attributes:
+      label: System Info
+      description: Please let us know your environment when reproducing the issue. 
+      placeholder: Operating System, Hilla version, Browsers type and version
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        If you think this ticket is important, add a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to help the community and maintainers prioritize this issue.
```
Comment (taefi) [inline .github/ISSUE_TEMPLATE/bug_report.yml:42]: Doesn't everyone think their tickets are important, especially when they put some time and effort to create a ticket? :) I think I understand the intention behind this, but probably we should reword it to communicate what we mean about the prioritization, or possibly to remove this from the Bug Report template.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional context
+      description: Is there anything else you can add about the proposal?
+      placeholder: additional context
+  - type: markdown
+    attributes:
+      value: |
+        If you think this ticket is important, add a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to help the community and maintainers prioritize this issue.
```
Comment (taefi) [inline .github/ISSUE_TEMPLATE/feature_request.yml:38]:
```suggestion
        If you think this feature is important, add a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to help the community and maintainers prioritize this issue.

```

**Review by taefi — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (tarekoraby) [reply .github/ISSUE_TEMPLATE/bug_report.yml:42]: > Doesn't everyone think their tickets are important

This message is not for the author of the ticket. It's for the people reading it. It is the same phrasing used by the docs: https://github.com/vaadin/docs/blob/latest/.github/ISSUE_TEMPLATE/doc_report.md

Comment (tarekoraby) [reply .github/ISSUE_TEMPLATE/bug_report.yml:42]: Another formulation could be the one used by this [project](https://github.com/aws/containers-roadmap/blob/master/.github/ISSUE_TEMPLATE/community-request-.md):

_Please vote on this issue by adding a 👍 reaction to the original issue to help the community and maintainers prioritize this request_

Comment (taefi) [reply .github/ISSUE_TEMPLATE/bug_report.yml:42]: I think this suggestion is way better than the one from docs. I believe it could be used on both issue and feature request templates.

**Review by taefi — APPROVED** [review]
